slack-base/src/main/java/com/hubspot/slack/client/models/teams/SlackTeamIF.java

```diff
@@ -13,6 +13,7 @@
 public interface SlackTeamIF {
   String getId();
   String getDomain();
+  String getName();
```

Comment:
I think this should be `Optional`. This client is by lots of projects so if we add a mandatory field, we must be ready to migrate all of them or make sure nobody does something like `SlackTeam.builder().setId("team_id").setDomain("domain").build()`. Otherwise, we are introducing backward-incompatible change.